src/common/k8sinstall/util.go

```diff
@@ -0,0 +1,113 @@
+package k8sinstall
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/openebs/openebs-e2e/apps"
+	"github.com/openebs/openebs-e2e/common"
+	"github.com/openebs/openebs-e2e/common/e2e_config"
+	"github.com/openebs/openebs-e2e/common/k8stest"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func CreateNamespace(namespace string) error {
+	cmd := exec.Command("kubectl", "create", "namespace", namespace)
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		logf.Log.Info("Error", "output", out)
+	}
+	return err
+}
+
+func InstallProduct() error {
+	err := installTheProduct()
+	return err
+}
+
+func installTheProduct() error {
+
+	var err error
+
+	_ = CreateNamespace(common.NSOpenEBS())
+
+	e2eCfg := e2e_config.GetConfig()
+	logf.Log.Info("e2e_config.GetConfig()", "CONFIG", e2eCfg)
+
+	_, err = k8stest.EnsureE2EAgent()
+	if err != nil {
+		return err
+	}
+
+	err = k8stest.GetNamespace(common.NSOpenEBS())
+	if err != nil {
+		return err
+	}
+
+	cmdArgs := []string{
+		"install",
+		e2eCfg.Product.OpenEBSHelmReleaseName,
+		"-n",
+		common.NSOpenEBS(),
+		e2eCfg.Product.OpenEBSHelmRepo,
+	}
+
+	err = apps.RemoveHelmRepository(e2e_config.GetConfig().Product.OpenEBSHelmRepoName, e2e_config.GetConfig().Product.OpenEBSHelmRepoUrl)
```

Comment:
Please add comments here describing why removing the helm repository and then adding it back again is necessary